cognite/neat/_graph/loaders/_rdf2dms.py

```diff
@@ -189,33 +187,12 @@ def _load(
             )
             yield _START_OF_CLASS(view.external_id)
             for identifier, properties in instance_iterable:
-                yield from self._create_instances(
-                    identifier, properties, projection, stop_on_exception, exclude=it.hierarchical_properties
-                )
-            if it.hierarchical_properties:
-                # Force the creation of instances, before we create the hierarchical properties.
-                yield _END_OF_CLASS
-                yield _START_OF_CLASS(f"{view.external_id} ({humanize_collection(it.hierarchical_properties)})")
-                yield from self._create_hierarchical_properties(it, projection, stop_on_exception)
-            if reader is not instance_iterable:
```

Comment:
We learned that hierarchical properties are not necessary to account for in the loading into DMS. Thus removing this from the loader.